cluster/gce/gci/configure-helper.sh

```diff
@@ -1688,8 +1688,8 @@ function start-kube-apiserver {
 
   if [[ -n "${ETCD_KMS_KEY_ID:-}" ]]; then
     ENCRYPTION_PROVIDER_CONFIG=$(cat << EOM | base64 | tr -d '\r\n'
-kind: EncryptionConfig
-apiVersion: v1
+kind: EncryptionConfiguration
+apiVersion: apiserver.config.k8s.io/v1
```

Comment:
is this api group in line with other componentconfigs? It's not really a componentconfig, or is it?

Reply:
@sttts I looked at the `kubelet.config.k8s.io` API group when I made this PR. How to ensure that this PR "in line with other componentconfigs"?

> It's not really a componentconfig, or is it?

Yes, it's need to set up API server for encrypting/decrypting resources when they are being read/write from/to database.

Reply:
That was my question, so `apiserver.config.k8s.io` is in line with it. Interestingly, the scheduler uses `componentconfig/v1alpha1`. That group is a big mess.